10.1/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -162,3 +165,63 @@ function wait_for_mysql_master() {
     sleep 1
   done
 }
+
+# Converts string version to the integer format (5.5 is converted to 505,
+# 10.1 is converted into 1001, etc.
+function version2number() {
+  local version=0
```

Comment:
Correct. mariadb uses version in format e.g. 10.0.12-MariaDB, so we need to remove the -MariaDB or any similar suffix as well. It might still be done better in bash though, so will try to fix..